src/test/java/com/microsoft/sqlserver/jdbc/fips/FipsTest.java

```diff
@@ -72,32 +71,12 @@ public void fipsEncryptTest() throws Exception {
         }
         catch (SQLServerException e) {
             Assertions.assertTrue(
-                    e.getMessage().contains("Could not enable FIPS due to either encrypt is not true or using trusted certificate settings."),
+                    e.getMessage().contains("Unable to verify FIPS mode settings."),
```

Comment:
I would suggest getting these same String error messages from SQLServerResource.java in test cases, such that your code becomes: `e.getMessage().contains(SQLServerResource.getResource("R_invalidFipsConfig")). `
It would remove all possibilities of errors and add maintainability.

Reply:
`getResource()` is not a public API

Reply:
Yeah.. I missed that, in that case we need a custom resource bundle for our test framework. I have created issue #466 to track this change.